applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_2D_beam_law.h

```diff
@@ -171,12 +171,12 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) LinearElastic2DBeamLaw : public GeoL
 
     void save(Serializer& rSerializer) const override
     {
-        KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, GeoLinearElasticPlaneStrain2DLaw)
+        KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, GeoIncrementalLinearElasticLaw)
```

Comment:
We could also have used the alias that keeps the base class instead of repeating the type here (i.e. less duplication):
```suggestion
        KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, BaseType)
```

Reply:
corrected